excel/beginner/charts/why-are-charts-useful.md

```diff
@@ -0,0 +1,42 @@
+---
+author: Stefan-Stojanovic
+
+levels:
+  - beginner
+
+aspects:
+  - new
+  - workout
+
+type: normal
+
+category: how to
+
+---
+
+# Why are charts useful?
+
+---
+## Content
+
+A good example of why charts are useful is because they can be used to detect patterns and understand results in a more clear way.
+
+They are good for understanding large quantities of data with ease and speed.
+
+A single chart can easily represent thousands of data.

```

Comment:
Combine this sentence with the one above it? Something like:

> They are good for understanding large quantities of data that would otherwise be difficult to individually analyze. A single chart can represent thousands of data points.